src/ashell/jerry-code.c

```diff
@@ -326,8 +326,17 @@ void restore_zjs_api() {
 #ifdef BUILD_MODULE_SENSOR
     zjs_sensor_init();
 #endif
+    jerry_value_t global_obj = jerry_get_global_object();
```

Comment:
@jimmy-huang https://github.com/01org/zephyr.js/pull/1044 will take care of this.